AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift

```diff
@@ -597,6 +621,29 @@ public class ConstructorIO: CIOSessionManagerDelegate {
             }
         }
     }
+    
+    private func executeQuizzes(_ request: URLRequest, completionHandler: @escaping QuizzesQueryCompletionHandler) {
```

Comment:
We'll need to break this out into `executeGetNextQuestion` and `executeGetQuizResults` to follow the change above. I wonder if it's possible to just have this be `executeQuizRequest` and have it handle both endpoints? 🤔